Maestro/subscriptions.json

```diff
@@ -1213,32 +1201,17 @@
     // Merge mirrored branches (to internal/) in dnceng Azure Devops
     {
       "triggerPaths": [
-        "https://github.com/aspnet/SignalR-Client-Cpp/blob/release/**/*",
-        "https://github.com/aspnet/AspLabs/blob/release/**/*",
-        "https://github.com/aspnet/AspNetWebStack/blob/release/**/*",
         "https://github.com/dotnet/aspnetcore/blob/release/2/**/*",
-        "https://github.com/dotnet/aspnetcore/blob/release/5.0/**/*",
         "https://github.com/dotnet/aspnetcore/blob/release/6.0/**/*",
-        "https://github.com/dotnet/aspnetcore/blob/release/7.0/**/*",
         "https://github.com/dotnet/aspnetcore/blob/release/8.0/**/*",
-        "https://github.com/aspnet/BrowserLink/blob/release/**/*",
-        "https://github.com/aspnet/BuildTools/blob/release/**/*",
```

Comment:
This one should stay

Reply:
For internal/* security servicing? The target branches don't exist.

Reply:
Yeah, there's an internal/release/2.1 branch for aspnet-buildtools